src/core/include/openvino/op/is_finite.hpp

```diff
@@ -25,6 +25,11 @@ class OPENVINO_API IsFinite : public Op {
     bool visit_attributes(AttributeVisitor& visitor) override;
 
     void validate_and_infer_types() override;
+
+    OPENVINO_SUPPRESS_DEPRECATED_START
+    bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
+    OPENVINO_SUPPRESS_DEPRECATED_END
+    bool has_evaluate() const override;
```

Comment:
Why do we need to have reference implementations for these operations? Do we have such ops in real models on constant branches?

Reply:
If I understand process correctly, adding reference implementation is requirement for OP enablement and it is also useful for plugin tests